Tests/ReactiveSwiftTests/SignalProducerLiftingSpec.swift

```diff
@@ -1666,5 +1666,47 @@ class SignalProducerLiftingSpec: QuickSpec {
 				expect(latestValues?.1) == 2
 			}
 		}
+		
+		describe("groupBy") {
```

Comment:
Sure I can do that. 
Should we check for all events that only this event is sent (e.g. when sending completed, also check that interrupted was not received), or can we trust that when `completed` is received, `interrupted` can not be received?